data/src/main/java/me/androidbox/data/local/agenda/AgendaLocalRepositoryImp.kt

```diff
@@ -73,6 +76,30 @@ class AgendaLocalRepositoryImp @Inject constructor(
         eventDao.deleteSyncEventsBySyncType(syncAgendaType)
     }
 
+    override suspend fun fetchAllRemindAtFromEvents(): List<Event> {
+        val eventEntity = eventDao.getAllRemindAt(System.currentTimeMillis())
```

Comment:
Should be plural right? Also I find `getAllRemindAt` a bit confusing together with `eventEntity`. The function name sounds like it returns remind at times, but apparently it returns event entities? 🤔